src/css/style/fixedDataTable.css

```diff
@@ -27,11 +27,17 @@
 }
 
 .public/fixedDataTable/header,
+.public/fixedDataTable/scrollbarSpacer,
 .public/fixedDataTable/header .public/fixedDataTableCell/main {
   background-color: var(--fbui-desktop-background-light);
   background-image: linear-gradient(#fff, #efefef);
 }
 
+.public/fixedDataTable/scrollbarSpacer {
+  position: absolute;
+  z-index: 99;
```

Comment:
why does it need this z-index?

Reply:
Because the scrollbar itself overlays the elements below it, so the spacer needs to live on the same plane. Otherwise, this will happen: 
![screen](https://user-images.githubusercontent.com/294747/33774060-a6ad8bd8-dbff-11e7-9d6b-a3cd0b6d0e5f.gif)

Reply:
ah cool, thanks for the image